package.json

```diff
@@ -9,16 +9,17 @@
   "license": "MIT",
   "main": "dist/app.js",
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "rimraf dist",
     "build": "yarn run clean && webpack-dashboard -- webpack --config configs/webpack.config.js --env.build --env.sourceMap",
     "dev": "webpack-dashboard -m -- webpack-dev-server --config configs/webpack.config.js --open",
     "dev2": "webpack-dashboard -- webpack-dev-server --config configs/webpack.config.js --open --env.awesome",
-    "lint": "tslint --project 'tsconfig.json'",
+    "lint": "tslint --project \"tsconfig.json\"",
```

Comment:
is this mandatory to work on windows? I prefer single quotes because they are clearer to read